nixpkgs_review/review.py

```diff
@@ -200,7 +200,15 @@ def build(self, packages: Set[str], args: str) -> List[Attr]:
     def build_pr(self, pr_number: int) -> List[Attr]:
         pr = self.github_client.pull_request(pr_number)
 
-        if self.use_ofborg_eval:
+        # keep up to date with `supportedPlatforms`
+        # https://github.com/NixOS/ofborg/blob/cf2c6712bd7342406e799110e7cd465aa250cdca/ofborg/src/outpaths.nix#L12
+        ofborg_platforms = [
+            "aarch64-darwin",
+            "aarch64-linux",
+            "x86_64-darwin",
+            "x86_64-linux",
```

Comment:
This is duplicated with https://github.com/Mic92/nixpkgs-review/blob/0f6760ba42bc9fa98a97764c0d3c45ce00b9ee76/nixpkgs_review/cli/__init__.py#L32-L35